doc/releases/quincy.rst

```diff
@@ -5,6 +5,305 @@ Quincy
 Quincy is the 17th stable release of Ceph.  It is named after Squidward
 Quincy Tentacles from Spongebob Squarepants.
 
+v17.2.4 Quincy
+==============
+
+This is the fourth backport release in the Quincy series. We recommend
+that all users update to this release.
+
+Notable Changes
+---------------
+
+* Cephfs: The 'AT_NO_ATTR_SYNC' macro is deprecated, please use the standard
+  'AT_STATX_DONT_SYNC' macro. The 'AT_NO_ATTR_SYNC' macro will be removed in
+  the future.
+
+* OSD: The issue of high CPU utilization during recovery/backfill operations
+  has been fixed. For more details see: https://tracker.ceph.com/issues/56530.
+
+* Trimming of PGLog dups is now controlled by size instead of the version.
+  This fixes the PGLog inflation issue that was happening when online
+  (in OSD) trimming jammed after a PG split operation. Also, a new offline
+  mechanism has been added: ``ceph-objectstore-tool`` now has a ``trim-pg-log-dups`` op
+  that targets situations where an OSD is unable to boot due to those inflated dups.
+  If that is the case, in OSD logs the "You can be hit by THE DUPS BUG" warning
+  will be visible.
+  Relevant tracker: https://tracker.ceph.com/issues/53729
+
+* OSD: Octopus modified the SnapMapper key format from
+  ``<LEGACY_MAPPING_PREFIX><snapid>_<shardid>_<hobject_t::to_str()>``
+  to
+  ``<MAPPING_PREFIX><pool>_<snapid>_<shardid>_<hobject_t::to_str()>``.
+  When this change was introduced, `94ebe0e <https://github.com/ceph/ceph/commit/94ebe0eab968068c29fdffa1bfe68c72122db633>`_
+  also introduced a conversion with a crucial bug which essentially
+  destroyed legacy keys by mapping them to
+  ``<MAPPING_PREFIX><poolid>_<snapid>_``
+  without the object-unique suffix. The conversion is fixed in this release.
+  Relevant tracker: https://tracker.ceph.com/issues/5614
+
+Changelog
+---------
+
+* .readthedocs.yml: Always build latest doc/releases pages (`pr#47442 <https://github.com/ceph/ceph/pull/47442>`_, David Galloway)
+* Add mapping for ernno:13 and adding path in error msg in opendir()/cephfs.pyx (`pr#46647 <https://github.com/ceph/ceph/pull/46647>`_, Sarthak0702)
+* admin: Fix check if PR or release branch docs build (`pr#47739 <https://github.com/ceph/ceph/pull/47739>`_, David Galloway)
+* backport doc/start: rewrite hardware-recs networks section (`pr#46652 <https://github.com/ceph/ceph/pull/46652>`_, Zac Dover)
+* backport of cephadm: fix osd adoption with custom cluster name (`pr#46551 <https://github.com/ceph/ceph/pull/46551>`_, Adam King)
+* backport: ceph-volume/tests: fix test_exception_returns_default (`pr#47435 <https://github.com/ceph/ceph/pull/47435>`_, Guillaume Abrioux)
```

Comment:
Ugh, I see one more tiny issue where these have been prefixed with "backport". Will fix it shortly.